src/pages/requestView/RequestSummary.tsx

```diff
@@ -1,146 +1,153 @@
 import React from "react";
-import { useParams } from "react-router-dom";
-import {
-  Request,
-  RequestWhere,
-  RequestSummaryQueryDocument,
-  Sample,
-  SampleMetadata
-} from "../../generated/graphql";
-import { useQuery } from "@apollo/client";
-
-function resolveLastUpdateDate(s: Sample) {
-  if (s.hasMetadataSampleMetadata.length == 1) {
-    return s.hasMetadataSampleMetadata[0];
-  }
+import {SortDirection, useRecentDeliveriesQuery, useRequestWithSamplesQuery} from "../../generated/graphql";
+import {AutoSizer, Column, InfiniteLoader, Table} from "react-virtualized";
+import {Row} from "react-bootstrap";
+import {observer} from "mobx-react";
+import "react-virtualized/styles.css";
+import _ from "lodash";
 
-  var smDataList: string[] = [];
-  for (var j = 0; j < s.hasMetadataSampleMetadata.length; j++) {
-    var sm = s.hasMetadataSampleMetadata[j];
-    smDataList.push(sm.importDate);
-  }
-  smDataList.sort();
-  var lastUpdate = smDataList[smDataList.length - 1];
-  for (var j = 0; j < s.hasMetadataSampleMetadata.length; j++) {
-    if (s.hasMetadataSampleMetadata[j].importDate == lastUpdate) {
-      return s.hasMetadataSampleMetadata[j];
+const RequestSummary = observer(({ props }) => {
+  const { loading, error, data, fetchMore } = useRequestWithSamplesQuery(
+    {
+      variables: {
+        where: {
+          igoRequestId: props.requestId
+        },
+        options: {
+          offset: 0,
+          limit: undefined
+        },
+      }
     }
-  }
-  return s.hasMetadataSampleMetadata[s.hasMetadataSampleMetadata.length - 1];
-}
-
-export interface IRequestSampleProps {
-  key?: string;
-  sample: Sample;
-}
-
-export class RequestSampleTableRecord extends React.Component<
-  IRequestSampleProps,
-  {}
-> {
-  latestSampleMetadata: SampleMetadata;
-  constructor(props: IRequestSampleProps) {
-    super(props);
-    this.latestSampleMetadata = resolveLastUpdateDate(props.sample);
-  }
+  );
+
+  if (loading) return <Row />;
+  if (error) return <Row>Error loading request details / request samples</Row>;
+
+  const request = data!.requests[0];
+  const samples = request.hasSampleSamples;
+
+
+
+  // function loadMoreRows({ startIndex, stopIndex }, fetchMore: any) {
+  //   return fetchMore({
+  //     variables: {
+  //       options: {
+  //         offset: startIndex,
+  //         limit: stopIndex
+  //       }
+  //     }
+  //   });
+  // }
+
+  // function isRowLoaded({ index }) {
+  //   return index < data.requests[-].length;
+  // }
```

Comment:
is this commented out for later?

if we're moving away from needing this then we can just remove it